src/libsync/propagatedownload.cpp

```diff
@@ -927,6 +927,9 @@ void PropagateDownloadFile::downloadFinished()
         if (!result) {
             done(SyncFileItem::NormalError, result.error());
             return;
+        } else if (result.get() == Vfs::ConvertToPlaceholderResult::Locked) {
+            done(SyncFileItem::SoftError, tr("The file %1 is currently in use").arg(_item->_file));
+            return;
         }
```

Comment:
Can we somehow enforce checking this better? 
So easy to miss one of the cases (null or != Ok).. :-\

Reply:
Good question.
The old solution for something like that would have been an error parameter.
But I'm really not sure that would enhance the situation...